### PR TITLE
5031 iot devops landing and takeover

### LINKED ISSUE #5031: [Iot DevOps WP] build takeover + landing page
[brief](https://docs.google.com/document/d/1imuKA06xisHD5RURurRhiJuoVeQ3qGojpOVX6odPxR0/edit?ts=5caa27c6)

### PR DESCRIPTION
**Pull request by solazio**
## Done

Add IoT DevOps WP takeover and landing page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/) and [/engage/iot-devops](http://0.0.0.0:8001/engage/iot-devops)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Make sure it complies with the [brief](https://docs.google.com/document/d/1imuKA06xisHD5RURurRhiJuoVeQ3qGojpOVX6odPxR0/edit?ts=5caa27c6#)
- Make sure it complies with the [design](https://github.com/canonical-web-and-design/www.ubuntu.com/issues/5023)

## Issue / Card

Fixes #5031

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -36,6 +36,7 @@
   {% include "takeovers/_openstack_security_takeover.html" %}
   {% include "takeovers/_14-04-esm.html" %}
   {% include "takeovers/_devops-iot-webinar_takeover.html" %}
```
Comment (pmahnke) [inline templates/index.html]: I am pretty sure we don't want both iot takeovers up at the same time... can you make sure?

File: templates/takeovers/_iot-devops-takeover.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    <div class="col-8">
+      <h1>The transformation of the DevOps journey in IoT</h1>
+      <p class="p-heading--four u-sv2">
+        Using Ubuntu Core and snaps to accelerate the Linux development cycle
+      </p>
+      <ul class="p-inline-list u-hide--small">
+        <li class="p-inline-list__item">
+          <!-- Link TBC -->
+          <a
+            href="/engage/openstack-security-webinar?utm_source=takeover&utm_campaign=CY19_DC_Openstack_WBN_OpenStackSecurity"
```
Comment (pmahnke) [inline templates/takeovers/_iot-devops-takeover.html]: bad link

File: templates/takeovers/_iot-devops-takeover.html [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+            class="p-button--positive"
+            ><span>Download the whitepaper</span></a
+          >
+        </li>
+        <li class="p-inline-list__item">
+          <a
+            href="/engage/openstack-security-whitepaper?utm_source=takeover&utm_campaign=CY19_DC_Openstack_Whitepaper_Security"
```
Comment (pmahnke) [inline templates/takeovers/_iot-devops-takeover.html]: bad link

File: templates/takeovers/_iot-devops-takeover.html [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+    <div class="col-4">
+      <p class="p-takeover__image u-align-text--center">
+        <img
+          src="{{ ASSET_SERVER_URL }}2c91289c-openstack-cloud.svg"
+          alt=""
+        />
+      </p>
+      <p class="u-hide--medium u-hide--large">
+        <a
+          href="/engage/openstack-security-webinar?utm_source=takeover&utm_campaign=CY19_DC_Openstack_WBN_OpenStackSecurity"
```
Comment (pmahnke) [inline templates/takeovers/_iot-devops-takeover.html]: bad link and CTA

File: templates/takeovers/_iot-devops-takeover.html [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+          class="p-button--positive"
+          ><span>Register for the webinar</span></a
+        >
+      </p>
+      <p class="u-hide--medium u-hide--large">
+        <a
```
Comment (pmahnke) [inline templates/takeovers/_iot-devops-takeover.html]: a second one? and bad

File: templates/engage/iot-devops.html [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+          A look at the technical architecture of Ubuntu Core and snaps in the context of an IoT DevOps model.         
+        </li>
+        <li class="p-list__item is-ticked">
+          Real life case studies of accelerated Linux development cycles as an alternative to the existing model. 
+        </li>
+      </ul>
+    </div>
+    <!-- Change this section when you have the info -->
+    <div class="col-5" id="register-section">
+      {% include "../shared/forms/_landing_page_default.html" with id="3348" returnURL="https://pages.ubuntu.com/LXD_Whitepaper_TY.html" %}
```
Comment (pmahnke) [inline templates/engage/iot-devops.html]: this is not the correct formid or thank you page

File: templates/takeovers/_iot-devops_takeover.html [hunk 7]
```diff
@@ -0,0 +1,53 @@
+<section class="p-strip--dark is-shallow p-takeover js-takeover">
+    <div class="row u-clearfix u-vertically-center">
+      <div class="col-7">
+        <h1>Transform the IoT DevOps journey</h1>
+        <p class="p-heading--four u-sv2">
+          Using Ubuntu Core and snaps to accelerate the Linux development cycle
+        </p>
+        <!-- Change link -->
+        <a
+          href="/engage/iot-devops"
```
Comment (caldav) [inline templates/takeovers/_iot-devops_takeover.html]:
```suggestion
          href="/engage/iot-devops?utm_source=takeover&utm_campaign=CY19_IOT_UbuntuCore_Whitepaper_IoTDevOps"
```

File: templates/takeovers/_iot-devops_takeover.html [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+        <p class="p-takeover__image u-align-text--center">
+          <img
+            src="{{ ASSET_SERVER_URL }}f3d518db-devops-iot.svg"
+            alt=""
+          />
+        </p>
+        <p class="u-hide--medium u-hide--large">
+          <!-- Change link -->
+          <a
+            href="/engage/iot-devops"
```
Comment (caldav) [inline templates/takeovers/_iot-devops_takeover.html]:
```suggestion
            href="/engage/iot-devops?utm_source=takeover&utm_campaign=CY19_IOT_UbuntuCore_Whitepaper_IoTDevOps"
```

File: templates/takeovers/_iot-devops_takeover.html [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+      <div class="col-7">
+        <h1>Transform the IoT DevOps journey</h1>
+        <p class="p-heading--four u-sv2">
+          Using Ubuntu Core and snaps to accelerate the Linux development cycle
+        </p>
+        <!-- Change link -->
+        <a
+          href="/engage/iot-devops"
+          class="p-button--positive u-hide--small" 
+          onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Homepage Link', 'eventAction' : 'Transform the IoT DevOps journey', 'eventLabel' : 'Download the whitepaper', 'eventValue' : undefined });"
```
Comment (caldav) [inline templates/takeovers/_iot-devops_takeover.html]: No need for inline onclick events on takeovers, they are already automatically generated.

File: templates/takeovers/_iot-devops_takeover.html [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+            src="{{ ASSET_SERVER_URL }}f3d518db-devops-iot.svg"
+            alt=""
+          />
+        </p>
+        <p class="u-hide--medium u-hide--large">
+          <!-- Change link -->
+          <a
+            href="/engage/iot-devops"
+            class="p-button--positive" 
+            onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Homepage Link', 'eventAction' : 'Transform the IoT DevOps journey', 'eventLabel' : 'Download the whitepaper', 'eventValue' : undefined });"
```
Comment (caldav) [inline templates/takeovers/_iot-devops_takeover.html]: No need for inline onclick events on takeovers, they are already automatically generated.

**Pull request merged**